packages/block-editor/src/components/block-list/block.js

```diff
@@ -144,13 +144,6 @@ function BlockListBlock( {
 		/>
 	);
 
-	// For aligned blocks, provide a wrapper element so the block can be
-	// positioned relative to the block column. This is enabled with the
-	// .is-block-content className.
-	if ( ! lightBlockWrapper && isAligned ) {
```

Comment:
https://github.com/WordPress/gutenberg/pull/21822/files#r413741147
In other words, would removing the check here not be enough?

Reply:
Discussed this in DM and not, this doesn't work